src/modules/QtQuick/ColorAnimation.js

```diff
@@ -0,0 +1,132 @@
+const EMPTY_ANIMATION_COLOR = new QmlWeb.QColor();
+
+QmlWeb.registerQmlType({
+  module: "QtQuick",
+  name: "ColorAnimation",
+  versions: /.*/,
+  baseClass: "PropertyAnimation",
+  properties: {
+    from: "color",
+    to: "color",
+  }
+}, class {
+  constructor(meta) {
+    QmlWeb.callSuper(this, meta);
+
+    this.$at = 0;
+    this.$loop = 0;
+
+    QmlWeb.engine.$addTicker((...args) => this.$ticker(...args));
+    this.runningChanged.connect(this, this.$onRunningChanged);
+  }
+  $redoActions() {
+    this.$actions = [];
+    for (let i = 0; i < this.$targets.length; i++) {
+      for (const j in this.$props) {
+        const target = this.$targets[i];
+        const property = this.$props[j];
+        const from_color = this.from;
+        const to_color = this.to;
+        // Animations always take time in SequentialAnimation
+        // , regardless of the value from and to
+        const action = {
+          // from, to
+          target,
+          property,
+          from_color,
+          to_color
+        };
+        this.$actions.push(action);
+      }
+    }
+    this.$startLoop();
+  }
+  $startLoop() {
+    for (const i in this.$actions) {
+      const action = this.$actions[i];
+      const from_color = action.from_color ||
+        action.target[action.property] ||
+        EMPTY_ANIMATION_COLOR;
+      const to_color = action.to_color ||
+        action.target[action.property] ||
+        EMPTY_ANIMATION_COLOR;
+
+      action.from = {
+        r: from_color.r,
+        g: from_color.g,
+        b: from_color.b,
+        a: from_color.a
+      };
+      action.to = {
+        r: to_color.r,
+        g: to_color.g,
+        b: to_color.b,
+        a: to_color.a
+      };
+    }
+    this.$at = 0;
+  }
+  $ticker(now, elapsed) {
+    if (!this.running && this.$loop !== -1 || this.paused) {
+      // $loop === -1 is a marker to just finish this run
+      return;
+    }
+    if (this.$at === 0 && this.$loop === 0 && !this.$actions.length) {
+      this.$redoActions();
+    }
+    this.$at += elapsed / this.duration;
+    if (this.$at >= 1) {
+      this.complete();
+      return;
+    }
+    for (const i in this.$actions) {
+      const action = this.$actions[i];
+      const property = action.target.$properties[action.property];
+      const progress = this.easing.$valueForProgress(this.$at);
+
+      const {
+        from,
+        to
+      } = action;
+
+      const cur_color = new QmlWeb.QColor(
+        from.r + (to.r - from.r) * progress,
+        from.g + (to.g - from.g) * progress,
+        from.b + (to.b - from.b) * progress,
+        from.a + (to.a - from.a) * progress
+      );
+
+      // Notification changes
+      property.set(cur_color, QmlWeb.QMLProperty.ReasonAnimation);
```

Comment:
I had try to use a special QColor instance for each animation only at the $startLoop called.
But something wrong happened, due to [QMLProperty.js](https://github.com/qmlweb/qmlweb/blob/master/src/engine/QMLProperty.js#L191-L207) will reset  value.
I do not know where the specific error, so take the most cost-effective way to achieve.
This does not cause performance problems for Modern browser. For performance optimization, we have to focus on reducing the operation of the DOM.

-----

This is an TestCode:
```qml
import QtQuick 2.0

Item {
    width: 300
    height: 300
    Rectangle {
        id: rect
        anchors.fill: parent
        color: "red"

        SequentialAnimation {
            id:ani
            running:false
            ColorAnimation { target: rect; property: "color"; from: 'red'; to: 'green'; duration: 2000 }
            ColorAnimation { target: rect; property: "color"; to: 'blue'; duration: 1000 }
       }
        Text{
            text:'<b>'+parent.color+'</b>'
            anchors.centerIn: parent
        }
        MouseArea{
            onClicked:console.log(ani.running = !ani.running)
            anchors.fill: parent;
        }
    }
}
```
and the Timeline info:
![image](https://cloud.githubusercontent.com/assets/2151644/21959002/a783dd5c-daf5-11e6-9347-f4d7ee9a69bb.png)

you see, the $updateImplicit spent a lot of time. That is what we should focus.